--- ref
+++ di
@@ -110,7 +110,6 @@
     DC_Y444
     Maximum TMDS clock: 300 MHz
     Extended HDMI video details:
-      Base EDID image size is in units of 1 cm
       HDMI VICs:
         HDMI VIC 1:  3840x2160   30.000000 Hz  16:9     67.500 kHz    297.000000 MHz
         HDMI VIC 3:  3840x2160   24.000000 Hz  16:9     54.000 kHz    297.000000 MHz
@@ -141,11 +140,6 @@
     Unique Rx, Ry: 0.62500000, 0.33203125
     Unique Gx, Gy: 0.30468750, 0.61328125
     Unique Bx, By: 0.15234375, 0.05078125
-  Vendor-Specific Audio Data Block (Dolby), OUI 00-D0-46:
-    Version: 1 (7 bytes)
-    Height speaker zone present
-    Surround speaker zone present
-    Center speaker zone present
   YCbCr 4:2:0 Capability Map Data Block:
     VIC  97:  3840x2160   60.000000 Hz  16:9    135.000 kHz    594.000000 MHz
     VIC 102:  4096x2160   60.000000 Hz 256:135  135.000 kHz    594.000000 MHz
@@ -156,22 +150,8 @@
     DTD 3:  1360x768    60.015162 Hz  85:48    47.712 kHz     85.500000 MHz
                  Hfront   64 Hsync 112 Hback  256 Hpol P
                  Vfront    3 Vsync   6 Vback   18 Vpol P
-Checksum: 0x4d  Unused space in Extension Block: 10 bytes
+Checksum: 0x4d
 
 ----------------
 
-Warnings:
-
-Block 1, CTA-861 Extension Block:
-  IT Video Formats are overscanned by default, but normally this should be underscanned.
-  Colorimetry Data Block: Set the sRGB colorimetry bit to avoid interop issues.
-
-Failures:
-
-Block 0, Base EDID:
-  Detailed Timing Descriptor #1: Mismatch of image size 800x450 mm vs display size 1220x680 mm.
-  Detailed Timing Descriptor #2: Mismatch of image size 800x450 mm vs display size 1220x680 mm.
-Block 1, CTA-861 Extension Block:
-  Detailed Timing Descriptor #3: Mismatch of image size vs display size: image size is not set, but display size is.
-
-EDID conformity: FAIL
+EDID conformity: PASS
