--- ref
+++ di
@@ -36,32 +36,9 @@
 
 Block 1, DisplayID Extension Block:
   Version: 2.0
-  Extension Count: 0
   Display Product Primary Use Case: Presentation display
-  Vendor-Specific Data Block (0x7e) (VESA), OUI 3A-02-92:
-    Data Structure Type: eDP
-    Default Colorspace and EOTF Handling: sRGB
-    Number of Pixels in Hor Pix Cnt Overlapping an Adjacent Panel: 0
-    Multi-SST Operation: Two Streams (number of links shall be 2 or 4)
-  Checksum: 0x61
-Checksum: 0x90  Unused space in Extension Block: 113 bytes
+Checksum: 0x90
 
 ----------------
 
-Warnings:
-
-Block 0, Base EDID:
-  Missing Display Product Name.
-
-Failures:
-
-Block 0, Base EDID:
-  Alphanumeric Data String: No newline, but one or more trailing spaces.
-  Alphanumeric Data String: No newline, but one or more trailing spaces.
-  Missing Display Range Limits Descriptor.
-EDID:
-  DisplayID: Missing DisplayID Display Parameters Data Block.
-  DisplayID: Missing DisplayID Type VII Detailed Timing Data Block.
-  DisplayID: DisplayID expects at least one preferred timing.
-
-EDID conformity: FAIL
+EDID conformity: PASS
