--- ref
+++ di
@@ -25,10 +25,10 @@
     DMT 0x2a:  1400x1050   59.978442 Hz   4:3     65.317 kHz    121.750000 MHz
     DMT 0x3a:  1680x1050   59.954250 Hz  16:10    65.290 kHz    146.250000 MHz
   Detailed Timing Descriptors:
-    DTD 1:  3840x2160   60.000000 Hz  16:9    135.000 kHz    594.000000 MHz (16 mm x 9 mm)
+    DTD 1:  3840x2160   60.000000 Hz  16:9    135.000 kHz    594.000000 MHz
                  Hfront  176 Hsync  88 Hback  296 Hpol P
                  Vfront    8 Vsync  10 Vback   72 Vpol P
-    DTD 2:  1920x1080   60.000000 Hz  16:9     67.500 kHz    148.500000 MHz (16 mm x 9 mm)
+    DTD 2:  1920x1080   60.000000 Hz  16:9     67.500 kHz    148.500000 MHz
                  Hfront   88 Hsync  44 Hback  148 Hpol P
                  Vfront    4 Vsync   5 Vback   36 Vpol P
     Display Product Name: 'Philips FTV'
@@ -96,33 +96,19 @@
     VIC  98:  4096x2160   24.000000 Hz 256:135   54.000 kHz    297.000000 MHz
     VIC 100:  4096x2160   30.000000 Hz 256:135   67.500 kHz    297.000000 MHz
   Video Format Preference Data Block:
-    VIC  97:  3840x2160   60.000000 Hz  16:9    135.000 kHz    594.000000 MHz
-    VIC  93:  3840x2160   24.000000 Hz  16:9     54.000 kHz    297.000000 MHz
-    VIC  95:  3840x2160   30.000000 Hz  16:9     67.500 kHz    297.000000 MHz
-    VIC  16:  1920x1080   60.000000 Hz  16:9     67.500 kHz    148.500000 MHz
+    VIC  97
+    VIC  93
+    VIC  95
+    VIC  16
   Detailed Timing Descriptors:
-    DTD 3:   720x480    59.940060 Hz   3:2     31.469 kHz     27.000000 MHz (16 mm x 9 mm)
+    DTD 3:   720x480    59.940060 Hz   3:2     31.469 kHz     27.000000 MHz
                  Hfront   16 Hsync  62 Hback   60 Hpol N
                  Vfront    9 Vsync   6 Vback   30 Vpol N
-    DTD 4:  1920x1080   24.000000 Hz  16:9     27.000 kHz     74.250000 MHz (16 mm x 9 mm)
+    DTD 4:  1920x1080   24.000000 Hz  16:9     27.000 kHz     74.250000 MHz
                  Hfront  638 Hsync  44 Hback  148 Hpol P
                  Vfront    4 Vsync   5 Vback   36 Vpol P
-Checksum: 0xc8  Unused space in Extension Block: 25 bytes
+Checksum: 0xc8
 
 ----------------
 
-Warnings:
-
-Block 1, CTA-861 Extension Block:
-  IT Video Formats are overscanned by default, but normally this should be underscanned.
-  YCbCr 4:2:0 Capability Map Data Block: Some YCbCr 4:2:0 timings are invalid for HDMI 2.1 (which requires an RGB timings pixel rate >= 590 MHz).
-  Add a Colorimetry Data Block with the sRGB colorimetry bit set to avoid interop issues.
-
-Failures:
-
-Block 1, CTA-861 Extension Block:
-  Missing VCDB, needed for Set Selectable RGB Quantization to avoid interop issues.
-EDID:
-  Base EDID: The DTD max image size is 16x9mm, but the display size is not specified anywhere.
-
-EDID conformity: FAIL
+EDID conformity: PASS
