--- ref
+++ di
@@ -110,7 +110,6 @@
   Colorimetry Data Block:
     xvYCC601
     xvYCC709
-    Gamut Boundary Description Metadata Profile P0
   Vendor-Specific Data Block (HDMI), OUI 00-0C-03:
     Source physical address: 4.0.0.0
     Supports_AI
@@ -141,33 +140,19 @@
     Full Frame Peak Luminance Index: 0
     Peak Luminance Index: 0
   Detailed Timing Descriptors:
-    DTD 3:  1920x1080i  50.000000 Hz  16:9     28.125 kHz     74.250000 MHz (1872 mm x 1053 mm)
+    DTD 3:  1920x540  i  50.044484 Hz  32:9     28.125 kHz     74.250000 MHz (1872 mm x 1053 mm)
                  Hfront  528 Hsync  44 Hback  148 Hpol P
-                 Vfront    2 Vsync   5 Vback   15 Vpol P Vfront +0.5 Odd Field
-                 Vfront    2 Vsync   5 Vback   15 Vpol P Vback  +0.5 Even Field
+                 Vfront    2 Vsync   5 Vback   15 Vpol P
     DTD 4:  1920x1080   60.000000 Hz  16:9     67.500 kHz    148.500000 MHz (1872 mm x 1053 mm)
                  Hfront   88 Hsync  44 Hback  148 Hpol P
                  Vfront    4 Vsync   5 Vback   36 Vpol P
-Checksum: 0x10  Unused space in Extension Block: 11 bytes
+Checksum: 0x10
 
 ----------------
 
-Warnings:
-
-Block 1, CTA-861 Extension Block:
-  Video Data Block: For improved preferred timing interoperability, set 'Native detailed modes' to 1.
-  Video Capability Data Block: Set Selectable YCbCr Quantization to avoid interop issues.
-  Colorimetry Data Block: Set the sRGB colorimetry bit to avoid interop issues.
-
 Failures:
 
-Block 0, Base EDID:
-  Detailed Timing Descriptor #1: Mismatch of image size 1872x1053 mm vs display size 1650x930 mm.
-  Detailed Timing Descriptor #2: Mismatch of image size 1872x1053 mm vs display size 1650x930 mm.
 Block 1, CTA-861 Extension Block:
-  Detailed Timing Descriptor #3: Mismatch of image size 1872x1053 mm vs display size 1650x930 mm.
-  Detailed Timing Descriptor #4: Mismatch of image size 1872x1053 mm vs display size 1650x930 mm.
-EDID:
-  Base EDID: The DTD max image size is 1872x1053mm, which is larger than the display size 1650.0x930.0mm.
-
+  Colorimetry Data Block: Reserved bits MD0-MD3 must be 0.
+  Vendor-Specific Video Data Block (HDR10+), OUI 90-84-8B: Peak luminance index 0 is reserved.
 EDID conformity: FAIL
