--- ref
+++ di
@@ -104,10 +104,9 @@
     DTD 2:  1920x1080   60.000000 Hz  16:9     67.500 kHz    148.500000 MHz (597 mm x 336 mm)
                  Hfront   88 Hsync  44 Hback  148 Hpol P
                  Vfront    4 Vsync   5 Vback   36 Vpol P
-    DTD 3:  1920x1080i  60.000000 Hz  16:9     33.750 kHz     74.250000 MHz (597 mm x 336 mm)
+    DTD 3:  1920x540  i  60.053381 Hz  32:9     33.750 kHz     74.250000 MHz (597 mm x 336 mm)
                  Hfront   88 Hsync  44 Hback  148 Hpol P
-                 Vfront    2 Vsync   5 Vback   15 Vpol P Vfront +0.5 Odd Field
-                 Vfront    2 Vsync   5 Vback   15 Vpol P Vback  +0.5 Even Field
+                 Vfront    2 Vsync   5 Vback   15 Vpol P
     DTD 4:  1280x720    60.000000 Hz  16:9     45.000 kHz     74.250000 MHz (597 mm x 336 mm)
                  Hfront  110 Hsync  40 Hback  220 Hpol P
                  Vfront    5 Vsync   5 Vback   20 Vpol P
@@ -117,31 +116,14 @@
     DTD 6:  1920x1080   50.000000 Hz  16:9     56.250 kHz    148.500000 MHz (597 mm x 336 mm)
                  Hfront  528 Hsync  44 Hback  148 Hpol P
                  Vfront    4 Vsync   5 Vback   36 Vpol P
-Checksum: 0xa6  Unused space in Extension Block: 10 bytes
+Checksum: 0xa6
 
 ----------------
 
-Warnings:
-
-Block 0, Base EDID:
-  Vendor & Product Identification: The serial number is one of the known dummy values, it should probably be set to 0.
-Block 1, CTA-861 Extension Block:
-  Video Data Block: VIC 16 and the first DTD are not identical. Is this intended?
-  Display Product Serial Number is set, so the Serial Number in the Base EDID should be 0.
-EDID:
-  Base EDID: Some timings are out of range of the Monitor Ranges:
-    Vertical Freq: 24.000 - 75.062 Hz (Monitor: 50.000 - 75.000 Hz)
-
 Failures:
 
 Block 0, Base EDID:
-  Basic Display Parameters & Features: sRGB is signaled, but the chromaticities do not match.
   Display Range Limits: Reserved bits of byte 14 are non-zero.
   Display Range Limits: Reserved bits of byte 15 are non-zero.
   Display Range Limits: Reserved bits of byte 16 are non-zero.
-Block 1, CTA-861 Extension Block:
-  Missing VCDB, needed for Set Selectable RGB Quantization to avoid interop issues.
-EDID:
-  CTA-861: Native progressive timings are a mix of several resolutions.
-
 EDID conformity: FAIL
